### PR TITLE
css-grid-classes | Cristofer Martinez

### PR DESCRIPTION
**Pull request by CristoferMar**
css-grid-classes

### CODE REVIEW

File: css-grid-classes/styles.css [hunk 1]
```diff
@@ -0,0 +1,24 @@
+body {
+  font-family: sans-serif;
+}
+
+.whole {
```
Comment (codingcodymiller) [inline css-grid-classes/styles.css:5]: You should name this class `container`

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start! In addition to my note in your code, you are also missing styling for the `one-fourth` class

**Comment by CristoferMar** [comment]
Updated container & added one-fourth

**Review by codingcodymiller — APPROVED** [review]
Great work!!

**Pull request merged**